db/week-1/lesson.md

```diff
@@ -29,6 +29,22 @@ Read the Mentors Notes [here](./mentors.md)
 
 ---
 
+## Learning Objectives
+
+By the end of this lesson students should be able to
+
+- Explain why databases are needed in Web Development
+- Define what SQL is and what it is used for
+- Describe what `table`, `rows` and `columns` are used for in a relational database
+- Install and create a database using PostgreSQL
+- Create a table in a database using PostgreSQL
+- Insert data into a table using PostgreSQL
+- Retrieve data from a table using PostgreSQL
+- Retrieve data from a table using conditionals in PostgreSQL
```

Comment:
Do we also want to cover ordering and/or aggregates and/or group-by? e.g. `SELECT * FROM t ORDER BY c LIMIT 1` and/or `SELECT MAX(c) FROM t` and/or `SELECT name, COUNT(*) FROM t GROUP BY name`

Reply:
Some of these may come in the week 2 class @illicitonion , it's not part of the first class.

Reply:
Yeah, I'm not sure _where_ these should be, but they're currently not listed in any of the three :) Week 2 probably makes sense

Reply:
I'm not sure why we're teaching CREATE TABLE in week 1 - the students have no experience of using tables at this stage so understanding what's needed to create one is much harder.  We also then have to teach SQL datatypes, etc.  Wouldn't it be better to concentrate on queries initially then progress to insert, update & delete later (e.g. week 2) and create table only after we've covered JOIN and primary/foreign key (maybe week 3).  I'm not sure how many front-end developers would ever need to use create table anyway.

Reply:
Hey :) At this stage, students have no experience whatsoever using SQL anyway. In Glasgow, in the past 2 years, we started with a paper exercise, by thinking about how to organise data for a problem so it was really nice to introduce the concepts of table, columns, rows, keys and types. Overall it worked really well, and it was one of the module where students had a lot of success to understand what's going on because everything is well structured in SQL. We are not teaching people to be frontend developer more than backend developer, at least in Glasgow, so i really think introducing  how things are stored first help a lot. If you start with insert, you still need to understand the concept of table, column and types, isn't it? So in my opinion, the most logical way is to actually start with creating tables :)